Yoast/wordpress-seo
# Make custom taxonomies available on search appearance

## Summary

This PR can be summarized in the following changelog entry:

* Custom taxonomy replacevariables are now visible on the search appearance settings page.

## Relevant technical choices:

* Like @Xyfi did for the custom fields in the search appearance, I've simulated the custom-taxonomy replacevars by appending an array of all custom_taxonomy names with `ct_` and `ct_desc_` appended. 

## Test instructions

This PR can be tested by following these steps:

* Go to the search appearance page.
* Press `%`.
* Verify that your custom fields are there.

## Quality assurance


Fixes #9999
